src/assets/i18n/en.json5

```diff
@@ -1516,6 +1516,7 @@
 
   "forgot-password.form.submit": "Submit password",
 
+  "password.error.robust-password": "New password is invalid. Valid passwords must be between 8-15 characters long and must include a minimum of: one uppercase letter, one lowercase letter, one number, and one special character (!?$@#$%^&+=).",
```

Comment:
After re-reviewing the backend PR (https://github.com/DSpace/DSpace/pull/8404/) I really wonder if it'd be easier to just define this message on the backend.  This exact same message *must* be returned by the 422 (PasswordNotValidException) on the backend, or else other clients will not be able to use this endpoint.

So, I wonder if we could simply remove this message and display the message returned in the 422?  Otherwise, we'll have to copy this exact same text into two locations...here on the frontend, and also in Messages.properties on the backend.

If we have to have it in two locations for now, that's OK, but we need to document it (likely in the backend configuration). Otherwise, sites which update the validation regex will not be aware that they need to change the text of this error in two places.

Reply:
Hi @tdonohue , yes, I agree with you. We already updated REST PR, we can change angular implementation in order to show the message returned in the 422 exception.